### PR TITLE
Fix broken links in README

### PR DESCRIPTION
**Pull request by sacha-l**
I _think_ that's where these hyperlinks are intended to link to.

### CODE REVIEW

**Review by bkchr — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ and treat as best.
 ## Collator
 
 A Polkadot [collator](https://wiki.polkadot.network/docs/en/learn-collator) for the parachain is
-implemented by [`cumulus-collator`](collator).
+implemented by [`cumulus-collator`](https://github.com/paritytech/cumulus/tree/master/client/collator).
```
Comment (bkchr) [inline README.md:31]: I think this was maybe more about the `polkadot-collator` binary.

File: README.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ beautiful and functional.
 
 ## Consensus
 
-[`cumulus-consensus`](consensus) is a
+[`cumulus-consensus`](https://github.com/paritytech/cumulus/tree/master/client/consensus) is a
```
Comment (sacha-l) [inline README.md:19]: Actually, shouldn't this be: https://github.com/paritytech/cumulus/blob/master/client/consensus/common/src/parachain_consensus.rs and renamed to `parachain-consensus` ?

File: README.md (hunk 1 above)
Comment (sacha-l) [reply README.md:31]:
```suggestion
implemented by the `polkadot-collator` binary.
```

File: README.md (hunk 2 above)
Comment (bkchr) [reply README.md:19]: Sounds sensible.

Comment (sacha-l) [inline README.md:19]:
```suggestion
[`parachain-consensus`](https://github.com/paritytech/cumulus/blob/master/client/consensus/common/src/parachain_consensus.rs) is a
```

**Pull request merged**